Fix rib classification and long bone definition

rays22 opened the pull request:
This commit intends to
1. Drop classification of `rib` under `long bone`. This is because there is a consensus in the field that ribs are not part of the `appendicular skeleton`.
2. Remove the restriction that long bones are only in the limbs from the `long bone` free text definition to make it consistent with the existing logical classification.
If applied, this commit will fix #2010.

dosumis commented:
Apologies for taking so long to submit review.  I always forget that there's one more button to click to submit all the comments I've added.

rays22 commented on src/ontology/uberon-edit.obo:
```diff
@@ -53380,7 +53379,7 @@ property_value: homology_notes "On the internal or parietal surface of the left
 name: long bone
-def: "Long bone is a limb bone that is subcylindrical and has a shaft with periosteum separating the ends of the bones. Long bones are present only in the limbs[VSAO:wd]." [https://github.com/obophenotype/uberon/issues/32, VSAO:wd, Wikipedia:Long_bone]
+def: "A long bone is a bone that has a shaft and 2 ends and is longer than it is wide. Long bones have a thick outside layer of compact bone and an inner medullary cavity containing bone marrow. The ends of a long bone contain spongy bone and an epiphyseal line. The epiphyseal line is a remnant of an area that contained hyaline cartilage that grew during childhood to lengthen the bone." [https:// ...
 comment: Endochondral_ossification is an essential process during the rudimentary formation of long bones, with the exception of the clavicle[Wikipedia:Endochondral_ossification]. The medial and and lateral ends undergo EO, the mid-portion is formed by a process with features of EO & IO (the process is shared by the mandible)[ISBN:9780397517251]
```
In the latest commit I have
- switched an `annotationProperty` from `comment` to `database_cross_reference`,
- changed the definition of `long bone` so that it works across species and across stages,
- changed the ticket references so that they use `seeAlso`.

rays22 commented:
In the latest commit I have
- switched an `annotationProperty` from `comment` to `database_cross_reference`,
- changed the definition of `long bone` so that it works across species and across stages,
- changed the ticket references so that they use `seeAlso`.

rays22 commented:
> In the latest commit I have
> 
> * switched an `annotationProperty` from `comment` to `database_cross_reference`,
> * changed the definition of `long bone` so that it works across species and across stages,
> * changed the ticket references so that they use `seeAlso`.

Hi @dosumis,
I think I have addressed your points. Could you take a look and check if indeed that is the case?
